packages/rn-tester/js/examples/Modal/ModalPresentation.js

```diff
@@ -45,11 +70,14 @@ function ModalPresentation() {
     React.useState('fullScreen');
   const [supportedOrientationKey, setSupportedOrientationKey] =
     React.useState('Portrait');
+  const [modalOpened, setModalOpened] = React.useState(null);
   const [currentOrientation, setCurrentOrientation] = React.useState('unknown');
   const [action, setAction] = React.useState('None');
+  let ref = React.useRef<?React.ElementRef<typeof Text>>(null);
   const actions = Platform.OS === 'ios' ? iOSActions : noniOSActions;
   const onDismiss = () => {
     setVisible(false);
+    setModalOpened(false);
```

Comment:
Is it possible to leverage visible state instead of creating a new state?

Reply:
@ryancat Thanks. I moved the example to the documentation https://github.com/facebook/react-native-website/pull/3438